docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'
```

Comment:
Recommend starting with line 5 and the bulleted list so that users know what this content is for rather than what it isn't for. Recommend moving line 3 to after the bullet line, so it would be inbetween current line 8 and line 9.